packages/components/src/components/link-tile-model-signature/link-tile-model-signature.examples.md

```diff
@@ -0,0 +1,188 @@
+can change the height by using different aspect ratios.
+
+<Playground :markup="aspectRatioMarkup">
+  <SelectOptions v-model="aspectRatio" :values="aspectRatios" name="aspectRatio"></SelectOptions>
+</Playground>
+
+---
+
+## Model
+
+<Playground :markup="modelMarkup">
+  <SelectOptions v-model="model" :values="models" name="model"></SelectOptions>
+</Playground>
+
+<script lang="ts">
+import Vue from 'vue';
+import Component from 'vue-class-component';
+import { TILE_WEIGHTS, TILE_ASPECT_RATIOS } from '../../utils'; 
+import { MODEL_SIGNATURE_MODELS } from '../model-signature/model-signature-utils'; 
+import { GROUP_DIRECTIONS } from '../../styles/group-direction-styles'; 
+
+@Component
+export default class Code extends Vue {
+  config = { spacing: 'block' };
+  img =  `<img src="${require('@/assets/image-grid.png')}" width="3000" height="2000" alt="Some alt text" />`;
+  primaryLink = '<p-link slot="primary" href="https://www.porsche.com">Primary label</p-link>';
+  secondaryLink = '<p-link slot="secondary" href="https://www.porsche.com">Secondary label</p-link>';
+
+  basic = `<p-link-tile-model-signature
+  heading="Some heading"
+>
+  ${this.img}
+  ${this.primaryLink}
+  ${this.secondaryLink}
+</p-link-tile-model-signature>
+<p-link-tile-model-signature
+  heading="Some heading"
+>
+  <picture>
+    <source media="(min-width:400px)" srcset="${require('@/assets/image-grid.png')}" />
+    ${this.img}
+  </picture>
+  ${this.primaryLink}
+  ${this.secondaryLink}
+</p-link-tile-model-signature>`;
+
+  weight = 'semi-bold';
+  weights = [...TILE_WEIGHTS, "{ base: 'semi-bold', m: 'regular' }"];
+  get weightMarkup() {
+    return`<p-link-tile-model-signature
+  heading="Some heading"
+  weight="${this.weight}"
+>
+  ${this.img}
+  ${this.primaryLink}
+  ${this.secondaryLink}
+</p-link-tile-model-signature>
+<p-link-tile-model-signature
+  heading="Some heading"
+  weight="${this.weight}"
+  description="Some description"
+>
+  ${this.img}
+  ${this.primaryLink}
+  ${this.secondaryLink}
+</p-link-tile-model-signature>`
+  };
+
+  description = `<p-link-tile-model-signature
+  heading="Some heading"
+  description="Some description"
+>
+  ${this.img}
+  ${this.primaryLink}
+  ${this.secondaryLink}
+</p-link-tile-model-signature>`;
+
+
+  linkDirection = 'row';
+  linkDirections = [...GROUP_DIRECTIONS, "{ base: 'row', m: 'column' }"];
+  get linkDirectionMarkup() {
+    return`<p-link-tile-model-signature
+  heading="Some heading"
+  link-direction="${this.linkDirection}"
+>
+  ${this.img}
+  ${this.primaryLink}
+  ${this.secondaryLink}
+</p-link-tile-model-signature>`
+  };
+
+  aspectRatio = '9:16';
+  aspectRatios = [...TILE_ASPECT_RATIOS, "{ base: '3:4', s: '1:1', m: '16:9' }"];
+  get aspectRatioMarkup() {
+    return`<p-link-tile-model-signature 
+heading="Some Heading"
+aspect-ratio="${this.aspectRatio}">
+  ${this.img}
+  ${this.primaryLink}
+  ${this.secondaryLink}
+</p-link-tile>`
```

Comment:
Opening and closing tags don't match always which leads to compilation errors in stackblitz.
<img width="841" alt="Bildschirm­foto 2023-03-21 um 18 35 15" src="https://user-images.githubusercontent.com/4380295/226694472-35782a4d-125a-4493-8e90-34a8a45acd33.png">